applications/dashboard/controllers/class.utilitycontroller.php

```diff
@@ -8,14 +8,44 @@
  * @since 2.0
  */
 
+use Garden\EventManager;
+use Gdn_Session as SessionInterface;
+use Vanilla\Contracts\ConfigurationInterface;
+use Vanilla\Web\Robots;
+
 /**
  * Handles /utility endpoint.
  */
 class UtilityController extends DashboardController {
 
+    /** Default robots.txt contents. */
+    private const ROBOTS_DEFAULT = <<<ROBOTS_DEFAULT
+User-agent: *
+Disallow: /entry/
+Disallow: /messages/
+Disallow: /profile/comments/
+Disallow: /profile/discussions/
+Disallow: /search/
+ROBOTS_DEFAULT;
+
```

Comment:
If we move this to core, shouldn't it be better for these hardcoded `const`s to live in `Vanilla\Web\Robots` class?

Reply:
My thoughts on this are:

1. They can't be private constants if they're on `Vanilla\Web\Robots`. Also, I'm _pretty sure_ they would only ever be used here. If I'm wrong and we do need to move them to `Robots`, having them as private constants means we can replace their usage without fear of breaking references in the future.
1. The contents of the default rules are application-specific URLs. I don't think they should live in general-purpose library classes. That said, the "invisible" rules could go in `Vanilla\Web\Robots`.